streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -1837,126 +1829,6 @@ public void shouldSetAdminClientTimeout() {
         assertThat(assignorConfiguration.getAdminClientTimeout(), is(2 * 60 * 1000));
     }
 
-    @Test
-    public void shouldRankPreviousClientAboveEquallyCaughtUpClient() {
```

Comment:
None of these tests are removed, just moved to `HighAvailabilityTaskAssignorTest`